### PR TITLE
Added overview.md & about.md 

### PR DESCRIPTION
**Pull request by Tonnu**
First version of the About page for the devcenter. The aim is to (briefly) describe wercker and why someone should use it.

### CODE REVIEW

File: content/docs/about.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#About wercker
+Wercker is a Docker-based platform for developing, building and
+delivering your applications.  Wercker provides you with a toolchain to speed up
+your development proces and allow you to build apps for the 21st century by
```
Comment (hatchan) [inline content/docs/about.md]: proces -> process

File: content/docs/about.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+containerising your code and automating your development pipelines.
```
Comment (hatchan) [inline content/docs/about.md]: containerising -> containerizing

File: content/docs/about.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+#Why wercker
+At wercker, we believe that micro-services and Service Oriented
+Architectures are the future. However, with great power comes great
+responsibility and micro-services naturally introduce a lot of extra moving
+parts in your application. To fully leverage the advantages of micro-services,
+they should be managed carefully.
+
+Wercker lets you do just that by setting up automated development pipelines.
+Very much in the spirit of "_release early, release often_" you can
+build and deploy your services with just a `git push` and let wercker work it's
```
Comment (hatchan) [inline content/docs/about.md]: it's -> its

File: content/docs/about.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+magic. 
+
+Under the hood wercker leverages Docker containers to not only provide users
+with isolated environments to run code in, but also to allow pipeline
+automatisation. Wercker runs your code in a container and takes it through what
+we call _steps_ - scripts [developed by the community](https://app.wercker.com/#explore) - and saves the
```
Comment (hatchan) [inline content/docs/about.md]: The middle part (` - scripts [developed by the community](https://app.wercker.com/#explore) - `) can be removed, imo

File: content/docs/about.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
+#About wercker
+Wercker is a Docker-based platform for developing, building and
+delivering your applications.  Wercker provides you with a toolchain to speed up
```
Comment (mies) [inline content/docs/about.md]: toolchain and cloud platform

File: content/docs/about.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+they should be managed carefully.
+
+Wercker lets you do just that by setting up automated development pipelines.
+Very much in the spirit of "_release early, release often_" you can
+build and deploy your services with just a `git push` and let wercker work it's
+magic. 
+
+Under the hood wercker leverages Docker containers to not only provide users
+with isolated environments to run code in, but also to allow pipeline
+automatisation. Wercker runs your code in a container and takes it through what
```
Comment (mies) [inline content/docs/about.md]: automatisation -> automation

File: content/docs/about.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+we call _steps_ - scripts [developed by the community](https://app.wercker.com/#explore) - and saves the
+output as a container.
+
+Wercker also works for developers working on a website, blog or app that is not
```
Comment (hatchan) [inline content/docs/about.md]: Pluralize website, blog and app. And is -> are

File: content/docs/about.md [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+necessaraly as complex. Because wercker is also a **workflow** and not only a
```
Comment (hatchan) [inline content/docs/about.md]: necessaraly -> necessaraly

File: content/docs/index.md [hunk 9]
```diff
@@ -1,6 +1,8 @@
-## Dive into specific topics
+## Overview 
+Welcome to the wercker devcenter! This site is where we document
+every feature of wercker and offer in-depth tutorials on various subjects. If
+this is your first time using wercker, we highly recommend starting with the
+[getting started]() guide.
```
Comment (hatchan) [inline content/docs/index.md]: Missing link

**Comment by hatchan** [comment]
Found a couple of small issue, needs some rebasing, and squashing. Other than that, lgtm

**Comment by Tonnu** [comment]
Implemented comments and opened a new PR against the overview branch so closing this one.

**Pull request closed without merging**